js/main.js

```diff
@@ -101,3 +104,66 @@ var addPinToDom = function () {
 
 addPinToDom();
 
+// Вернуть тип отеля для карточки отеля с описанием
+var getRoomType = function (objType) {
+  switch (objType) {
+    case 'flat':
+      return 'Квартира';
+    case 'bungalo':
+      return 'Бунгало';
+    case 'house':
+      return 'Дом';
+    default:
+      return 'Дворец';
+  }
+};
+
+// Создать шаблон card
+var createMapCard = function (obj) {
+  var mapFilterContainerNode = document.querySelector('.map__filters-container');
+  var cardNode = document.querySelector('#card').content;
+  var popup = cardNode.querySelector('.popup').cloneNode(true);
+  var popupTitle = popup.querySelector('.popup__title');
+  var popupAddress = popup.querySelector('.popup__text--address');
+  var popupPrice = popup.querySelector('.popup__text--price');
+  var popupType = popup.querySelector('.popup__type');
+  var popupCapacity = popup.querySelector('.popup__text--capacity');
+  var popupTime = popup.querySelector('.popup__text--time');
+  var popupFeatures = popup.querySelector('.popup__features');
+  var popupDescription = popup.querySelector('.popup__description');
+  var popupPhotos = popup.querySelector('.popup__photos');
+  var popupPhotosImg = popupPhotos.querySelector('img');
+  var popupAvatar = popup.querySelector('.popup__avatar');
+
+  popupTitle.textContent = obj.offer.title;
+  popupAddress.textContent = obj.offer.address;
+  popupPrice.textContent = obj.offer.price + '₽/ночь';
+  popupType.textContent = getRoomType(obj.offer.type);
```

Comment:
Проще и производительнее создать объект и забирать значение из него
```suggestion
  popupType.textContent = ROOM_TYPES[obj.offer.type];
```